pkg/agent/openflow/client_test.go

```diff
@@ -2917,3 +2917,30 @@ func TestSubscribeOFPortStatusMessage(t *testing.T) {
 	bridge.EXPECT().SubscribePortStatusConsumer(ch).Times(1)
 	c.SubscribeOFPortStatusMessage(ch)
 }
+
+func Test_client_InstallL7NetworkPolicyFlows(t *testing.T) {
```

Comment:
Better to make the test name consistent with other test name. `TestInstallL7NetworkPolicyFlows`

Reply:
I'm following the style of other tests in this file, like `Test_client_RegisterPacketInHandler`.